### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by bergr7**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
```
Comment (sariola) [inline .github/ISSUE_TEMPLATE/bug_report.md:28]: Maybe we want to recheck these. This and the below mobile device details would seem more applicable for a frontend application! Same with the reproduce steps above. What do you think?

Comment (bergr7) [reply .github/ISSUE_TEMPLATE/bug_report.md]: > Maybe we want to recheck these. This and the below mobile device details would seem more applicable for a frontend application! Same with the reproduce steps above. What do you think?

Agreed. Please check new ones.

Comment (sariola) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Solid. Extra nice to have the other templates too!

Comment (sariola) [reply .github/ISSUE_TEMPLATE/bug_report.md]: We could think of creating a guide to adding architectures.

**Review by sariola — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (bergr7) [reply .github/ISSUE_TEMPLATE/bug_report.md]: > We could think of creating a guide to adding architectures.

yeah, we need a bit more clarity into the process for other architectures but possible.

**Pull request merged**